src/main/java/edu/hawaii/its/api/service/OotbActiveUserProfileService.java

```diff
@@ -0,0 +1,74 @@
+package edu.hawaii.its.api.service;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import jakarta.annotation.PostConstruct;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import edu.hawaii.its.api.type.OotbActiveUserProfile;
+
+@Service
+public class OotbActiveUserProfileService {
+
+    @Autowired
+    private OotbGroupingPropertiesService ootbGroupingPropertiesService;
+    private Map<String, OotbActiveUserProfile> users = new HashMap<>();
+    private String ootbActiveUserProfile = "ADMIN";
+
+    @PostConstruct void init() {
+        initUsers();
+        updateActiveDefaultUser(ootbActiveUserProfile);
+    }
+
+    private void initUsers() {
+        users.put("USER", new OotbActiveUserProfile.Builder()
+                .uid("user0123")
+                .uhUuid("11111111")
+                .authorities(List.of("ROLE_UH"))
+                .cn("user")
+                .name("DefaultUser")
+                .email("user@hawaii.edu")
+                .build());
```

Comment:
Michael mentioned this in our last meeting, USER should be renamed to MEMBER. Make sure to update this throughout the code and in your UI PR too, thanks!